File: parallel-consumer-proxy-clients/parallel-consumer-proxy-client-kotlin/src/main/kotlin/bz/stub/parallelconsumer/client/coroutines/HelloFixture.kt
// Copyright (C) 2026 Antony Stubbs and contributors

package bz.stub.parallelconsumer.client.coroutines

/**
 * Prints the one line every language in this tree prints - the Kotlin end of the polyglot build
 * scaffolding (astubbs#242). The wording is fixed by `bin/foreign-client-step.sh`; a change there
 * has to change all eleven.
 */
public object HelloFixture {

    public const val LINE: String = "parallel-consumer-proxy-client hello fixture: kotlin"

    @JvmStatic
    public fun main(args: Array<String>) {
        print(LINE)
    }
}

File: parallel-consumer-proxy-clients/parallel-consumer-proxy-client-scala/src/main/scala/bz/stub/parallelconsumer/client/scaladsl/HelloFixture.scala
// Copyright (C) 2026 Antony Stubbs and contributors

package bz.stub.parallelconsumer.client.scaladsl

/** Prints the one line every language in this tree prints - the Scala end of the polyglot build
  * scaffolding (astubbs#242). The wording is fixed by `bin/foreign-client-step.sh`; a change there
  * has to change all eleven.
  */
object HelloFixture {

  val Line: String = "parallel-consumer-proxy-client hello fixture: scala"

  def main(args: Array[String]): Unit = print(Line)
}
